app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryInit.kt

```diff
@@ -20,7 +20,7 @@ class CustomDrugEntryInit : Init<CustomDrugEntryModel, CustomDrugEntryEffect> {
   ): First<CustomDrugEntryModel, CustomDrugEntryEffect> {
     val updatedModel = model.drugNameLoaded(drugName)
 
-    val effects = setOf(SetSheetTitle(drugName, null, null), SetDrugFrequency(null))
+    val effects = setOf(SetSheetTitle(drugName, null, null), SetDrugFrequency(null), SetDrugDosage(null))
```

Comment:
Because, setting frequency as null in this case will mean that setting it to `None` And as we are not rendering this from the ui. I am calling it from here.

Reply:
I don't need to `SetDrugDosage` here tho

Reply:
hmm, can't we just have the default text as `None`? That way it only changes we are passing the actual frequency value.

Reply:
Umm fair enough